### ISSUE TITLE
libspeex vs. libspeexdsp warning in Qt 5.5.0

### ISSUE THREAD

**Issue by mkrautz** [mumble-voip/mumble#1817]
```
C:\Users\mkrautz\mumble>qmake -recursive main.pro CONFIG+="release static no-g15 no-asio no-elevation"
Reading C:/Users/mkrautz/mumble/src/mumble_proto/mumble_proto.pro
Reading C:/Users/mkrautz/mumble/3rdparty/speex-build/speex-build.pro
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/fftwrap.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/fftwrap.c
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/kiss_fft.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/kiss_fft.c
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/kiss_fftr.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/kiss_fftr.c
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/smallft.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/smallft.c
WARNING: Automatically turning off nmake's inference rules. (CONFIG += no_batch)
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/fftwrap.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/fftwrap.c
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/kiss_fft.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/kiss_fft.c
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/kiss_fftr.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/kiss_fftr.c
WARNING: C:/Users/mkrautz/mumble/3rdparty/speex-src/libspeex/smallft.c conflicts with C:/Users/mkrautz/mumble/3rdparty/speexdsp-src/libspeexdsp/smallft.c
WARNING: Automatically turning off nmake's inference rules. (CONFIG += no_batch)
Reading C:/Users/mkrautz/mumble/3rdparty/celt-0.7.0-build/celt-0.7.0-build.pro
Reading C:/Users/mkrautz/mumble/3rdparty/celt-0.11.0-build/celt-0.11.0-build.pro
Reading C:/Users/mkrautz/mumble/3rdparty/opus-build/opus-build.pro
Reading C:/Users/mkrautz/mumble/3rdparty/minhook-build/minhook-build.pro
Reading C:/Users/mkrautz/mumble/src/mumble/mumble.pro
[...]
```

**Comment by mkrautz** [comment]
Functionally, everything is the same. Build works.

**Comment by hacst** [comment]
What does it mean with "conflicts"? Do would they generate equivalently named object files in the same target directory or something?

**Comment by mkrautz** [comment]
I believe it means:

fftwrap.c is in SOURCES.

VPATH is set to both speex-src/libspeex and speexdsp-src/libspeexdsp.

So fftwrap.c in SOURCES is ambiguous.

I'll pick one.

**Comment by davidebeatrici** [comment]
Fixed by ca8a117fae1182d873176030b558917a1acfdf13.
